### ISSUE TITLE
error provider in view transaction_form_clicandpay

### ISSUE THREAD

**Issue by romainsady** [lyra/plugin-odoo#12]
Hello,

There is a little oversight in the view: transaction_form_clicandpay
Indeed in version 10 of odoo provider is unknown ...

Example: We should add:
payment_views.xml

```
...
<field name="acquirer_reference" position="after">
    <field name="provider"/>
...
```

and in payment.py

```
...
class TransactionClicandpay(models.Model):
    _inherit = 'payment.transaction'
...
    provider = fields.Char(compute='_compute_provider')

    @api.one
    def _compute_provider(self):
        self.provider =  self.acquirer_id.provider
...
```

thus in the accounting part, payment transactions are accessible :)

sorry for my English

**Comment by nabil509** [comment]
Hi @SecretInformatique ,

Sorry for the delay.

Thank you for reporting. We will check this and try to fix it for the next version.

**Comment by nabil509** [comment]
Hi @SecretInformatique 

Error fixed for Odoo 10-14 in our plugin version 1.2.2. https://github.com/lyra/plugin-odoo/releases/tag/v1.2.2

Thank you for reporting. I close issue.
